### ISSUE TITLE
show title issues in Bootstrap

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#1024]
Not sure if this has always been an issue or whether it's something that has changed or indeed if it's deliberate for some reason. But it was pointed out to me today that the show title option on a text element inside a bootstrap accordion navigator doesn't work. e.g. it doesn't show that title whether ticked or not. I did some testing and it's a bit more nuanced than that e.g. the nav set to pills or tabs has a different impact to the nav set to accordion or carousel.

To reproduce add a text element and image element to the page with show title enabled - the title as well as content of both elements shows correctly.
Create a navigator below those elements set to accordion and either duplicate or recreate a text and image element with show title enabled inside that accordion pane the title of those elements inside the accordion do not show. 
Change the navigator to pills or tabs the titles do now show but inside the navigator much smaller than outside the navigator. That may just be a css issue and may be desirable that the sizes are different inside and outside the navigator.

To summarise the titles don't show in accordion or carousel but do show in pills or tabs but different size to when outside the navigator.

I think we should change this so that if the show title option is checked then the title should show regardless of navigator mode.
Not sure exactly what we should do about the difference in title size inside or outside the navigator and that may vary by theme anyway.

**Comment by obgeneralao** [comment]
Hi Ron,

I just updated to the latest devel and I still encounter this issue in the bootstrap sidebar navigation since long before.

![Xerte_1](https://user-images.githubusercontent.com/13013814/120896369-c7b44f00-c653-11eb-8af6-c9c6eba8507f.gif)

**Comment by ronm123** [comment]
Hi all
so I've investigated the various issues with this and have a solution but having just discussed with Tom it's something we are going to delay until the next dev day because there are implications for existing projects. e.g. if we change the code so that child titles are now showing (regardless of navigator mode) where they weren't before then we potentially have lots of unwanted titles showing. Also I think we need to change from <p><b> to h3 which again could have implications for existing projects. But we have a plan to avoid any consequences for existing projects which we'll discuss on the next dev day.
